lib/explorer/data_frame.ex

```diff
@@ -129,17 +129,65 @@ defmodule Explorer.DataFrame do
 
   @doc """
   Reads a IPC file into a dataframe.
+
+    ## Options
+
+      * `columns` - List with name of columns to be selected. Defaults to all columns.
+      * `projection` - List with the index of columns to be selected.
```

Comment:
Does it defaults to all columns as well?

Reply:
Yes, it does. I'll add this information.